GherkinSyncTool.Synchronizers.AzureDevOps/Model/TestParameters.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections.Generic;
+using System.Xml.Serialization;
+
+namespace GherkinSyncTool.Synchronizers.AzureDevOps.Model
+{
+    [XmlRoot(ElementName = "param")]
+    public class Param
```

Comment:
Seems like this class should be moved to separate file

Reply:
The classes are very small and related let them be together?